handsontable/src/plugins/formulas/formulas.js

```diff
@@ -614,7 +644,35 @@ export class Formulas extends BasePlugin {
       return false;
     }
 
-    return this.engine.getFillRangeData(engineSourceRange, engineTargetRange);
+    const fillRangeData = this.engine.getFillRangeData(engineSourceRange, engineTargetRange);
+    const [sourceStartRow, sourceStartColumn] = [engineSourceRange.start.row, engineSourceRange.start.col];
+    const [sourceEndRow, sourceEndColumn] = [engineSourceRange.end.row, engineSourceRange.end.col];
+    const [populationRowLength, populationColumnLength] = [sourceEndRow - sourceStartRow + 1,
+      sourceEndColumn - sourceStartColumn + 1];
```

Comment:
I'd simplify the code to use object destructing and just simple variable assigning. Now, the code looks overcomplicated.
```suggestion
    const {
      row: sourceStartRow,
      col: sourceStartColumn,
    } = engineSourceRange.start;
    const {
      row: sourceEndRow,
      col: sourceEndColumn,
    } = engineSourceRange.end;
    const populationRowLength = sourceEndRow - sourceStartRow + 1;
    const populationColumnLength = sourceEndColumn - sourceStartColumn + 1;
```

Reply:
I did it intentionally to shorten the code, but I don't mind changing it.